cmd/manager/main.go

```diff
@@ -37,12 +37,20 @@ import (
 func main() {
 	klog.InitFlags(nil)
 	flag.Set("logtostderr", "true")
+	watchNamespace := flag.String("namespace", "",
```

Comment:
Should we define this outside the main function instead?

Reply:
If we had more cli flags, it would make sense to create a `CliFlags` struct and parse the flags in a separate function that would return the `CliFlags` struct. Considering we have just one flag other than the `logtostderr` this should be good.